excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -232,53 +249,119 @@ public static String getCellValue(Workbook wb, Cell cell) {
             throw new IllegalStateException("Unknown cell type: " + cell.getCellType());
         }
 
-        logger.debug("cell {} resolved to value: {}", cellCoordinate, result);
+        logger.debug("cell ({},{}) resolved to value: {}", cell.getRowIndex(), cell.getColumnIndex(), result);
 
         return result;
     }
 
-    private static String getFormulaCellValue(Workbook wb, Cell cell) {
+    private static Object getErrorResult(final Cell cell) {
```

Comment:
This method always returns a `String` (unless it throws an Exception of course), so why not change the return signature to `String`?

Reply:
true, I'll change